canvas_modules/common-canvas/src/common-properties/controls/dropdown/dropdown.jsx

```diff
@@ -279,7 +280,8 @@ class DropDown extends React.Component {
 					translateWithId={(id) => listBoxMenuIconTranslationIds[id]}
 					titleText={this.props.controlItem}
 					helperText={this.props.control.helperText}
-					shouldFilterItem={this.filterItems}
+					allowCustomValue
```

Comment:
Not sure why this isnt required as a default for ComboBox, but if not set, it was causing an infinite loop when user clears the value while the dropdown is opened. Since custom values weren't allowed, it would automatically reset after losing focus and the onChange callback was kept getting called

https://github.com/user-attachments/assets/52d16d45-942a-49aa-83c3-24f5a4eb0f67